README.md

```diff
@@ -21,33 +21,53 @@ More advanced users, or those who are interested in using the latest pre-release
 The following sections must be followed prior to building the Morpheus container or building Morpheus from source.
 
 #### Requirements
-- Pascal architecture or better
+- Pascal architecture GPU or better
 - NVIDIA driver `450.80.02` or higher
 - [Docker](https://docs.docker.com/get-docker/)
 - [The NVIDIA container toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker)
+- [NVIDIA Triton Inference Server](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tritonserver) `22.02` or higher
```

Comment:
Based on #295 , this should probably be 22.06 now due to the somewhat strict compatibility requirements for Treelite in the ransomware models